cli/dcos-spark/submit_builder_test.go

```diff
@@ -21,6 +21,7 @@ const keytabPrefixed = "__dcos_base64__keytab"
 const keytab = "keytab"
 const sparkAuthSecret = "spark-auth-secret"
 const marathonAppId = "spark-app"
+const mesosSandbox = "/mnt/mesos/sandbox"
```

Comment:
`/mnt/mesos/sandbox` is local to a container and can potentially vary. Can we use `$MESOS_SANDBOX` environment variable which is set by executor to avoid binding to an absolute path?

Reply:
Since our CLI (`submit_builder.go`) is not using `$MESOS_SANDBOX` and hard coding that value, we have to do the same for the test also.
Will `$MESOS_SANDBOX` be available everywhere, where CLI is running?